app/views/users/new.html.haml

```diff
@@ -1,7 +1,13 @@
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+
+<script type="text/javascript">Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");</script>
+
+= javascript_include_tag 'payment'
```

Comment:
I am going to leave this as is, for the reason that I would prefer this js only to be loaded when the new user template is visited.  Is that a good practice?